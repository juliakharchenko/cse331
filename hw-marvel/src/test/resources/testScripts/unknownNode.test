# Tests finding a path to and from a node that doesn't exist within the graph

LoadGraph nba nbaPlayers.csv

FindPath nba KOBE-BRYANT DWIGHT-HOWARD

FindPath nba SHAQUILLE-ONEAL MAGIC-JOHNSON

FindPath nba DWIGHT-HOWARD SHAQUILLE-ONEAL