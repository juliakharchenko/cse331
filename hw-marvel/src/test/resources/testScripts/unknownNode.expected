# Tests finding a path to and from a node that doesn't exist within the graph

loaded graph nba

unknown: DWIGHT-HOWARD

unknown: SHAQUILLE-ONEAL

unknown: DWIGHT-HOWARD
unknown: SHAQUILLE-ONEAL